### PR TITLE
RET-2053: Update ave Nov 2017

### PR DESCRIPTION
**Pull request by cjoe**

### CODE REVIEW

File: deps.sh [hunk 1]
```diff
@@ -10,49 +10,37 @@ source $SCRIPTPATH/config.sh
 
 if [ "$MOS" == "OSX" ]; then
 
-# disabling fink!
-# some problems with sierra + general unhappiness with slowness/flakiness
-# stand by as we migrate away
-# source $SCRIPTPATH/os/osx/fink.sh
+    # disabling fink!
```
Comment (jeffditullio) [inline deps.sh]: what's do these comments mean? are they even relevant anymore?

File: pkgs/available/python.sh [hunk 2]
```diff
@@ -56,9 +56,8 @@ $PIP install versiontools  # ssl cert validation fail when installed in gevent-s
 $PIP install git+https://github.com/abourget/gevent-socketio.git
 
 $PIP install git+https://github.com/benoitc/gunicorn.git
-$PIP install git+https://github.com/mattbillenstein/flask-classy.git
+$PIP install flask-classful
```
Comment (jeffditullio) [inline pkgs/available/python.sh]: what does flask-classy/ful do?

**Comment by cjoe** [comment]
I am waiting for @honeybeep before I update ave because I am wondering how I will impact things outside of automation.

**Comment by honeybeep** [comment]
@cjoe this looks good to me! I doubt we were actually using flask for anything, it's probably just a (by now outdated) nice to have on the base image. I think you should just land this and push. I am backing up the Ubuntu directory in any case -- logged into netops and ran 

sudo -u ve rsync -av ve/Ubuntu ve/Ubuntu.BAK

from the ve home dir

**Pull request merged**